composer.json

```diff
@@ -18,7 +18,7 @@
         }
     },
     "require": {
-        "phpunit/phpunit": "^7.0 | ^8.0",
+        "phpunit/phpunit": "^7.0|^8.0|^9.0",
         "symfony/expression-language": "^3.3"
```

Comment:
Thanks, this change looks good.

It would be good to support newer versions of symfony/expression-language - the latest one is v5.0.5

Reply:
#7